src/site/content/en/blog/components-howto-tabs/index.md

```diff
@@ -121,6 +121,12 @@ Event handlers that are not attached to this element need to be bound if they ne
       this._onSlotChange = this._onSlotChange.bind(this);
 ```
 
+For progressive enhancement, the markup should alternate between tabs and panels. Elements that reorder their children tend to not work well with frameworks. Instead shadow DOM is used to reorder the elements by using slots.
+
+```js
+      this.attachShadow({ mode: 'open' });
+```
```

Comment:
See the [source](https://github.com/GoogleChromeLabs/howto-components/blob/079d0fa34ff9038b26ea8883b1db5dd6b677d7ba/elements/howto-tabs/howto-tabs.js#L67-L71).